logging/log_api.go

```diff
@@ -147,8 +157,8 @@ func (level LogLevel) String() string {
 		return "fatal"
 	case ErrorLevel:
 		return "error"
-	case WarnLevel:
-		return "warn"
+	case WarningLevel:
```

Comment:
To avoid breaking changes and be more consistent with [logrus levels](https://godoc.org/github.com/sirupsen/logrus#Level) I would suggest to not change this.